src/useRendersCount/__tests__/dom.ts

```diff
@@ -0,0 +1,22 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useRendersCount } from '../useRendersCount';
+
+describe('useRendersCount', () => {
+  it('should be defined', () => {
+    expect(useRendersCount).toBeDefined();
+  });
+
+  it('should return number', () => {
+    expect(renderHook(() => useRendersCount()).result.current).toEqual(expect.any(Number));
+  });
```

Comment:
This test is unnecessary, since TypeScript guarantees that a number is returned.